src/loader.js

```diff
@@ -44,22 +44,31 @@ const MiniCssExtractPlugin = require("./index");
  * @returns {string}
  */
 function hotLoader(content, context) {
-  const accept = context.locals
-    ? ""
-    : "module.hot.accept(undefined, cssReload);";
-
+  const localsJsonString = JSON.stringify(JSON.stringify(context.locals));
```

Comment:
Why double `JSON.stringify`? Just typo?

Reply:
`const localsJsonString = ${localsJsonString};`

string comparation is faster